mobile/src/components/chat/components/conversations-list/components/conversation/conversation.tsx

```diff
@@ -56,7 +62,7 @@ const Conversation: FC<Props> = ({
           {chatOpponent.userDetails.fullName}
         </Text>
         {lastMessageDate && (
-          <Text style={styles.opponentMessage}>{chatLastMessage}</Text>
+          <Text style={styles.opponentMessage}>{message}</Text>
```

Comment:
I don't know for sure how it works - but won't you'd have `ellipsizeMode` and `numberOfLines` from Text for this purpose?

Reply:
Yes, you are right, forget about this)